js/search/utils/checkDiskSpace.js

```diff
@@ -25,13 +25,21 @@ function checkDiskSpace(path, resolve, reject) {
             return resolve(space >= searchConfig.MINIMUM_DISK_SPACE);
         });
     } else {
-        exec(`fsutil volume diskfree ${path}`, (error, stdout, stderr) => {
+        exec(`fsutil volume diskfree ${path}`, (error, stdout) => {
             if (error) {
-                if (stderr.indexOf("No such file or directory") !== -1) {
+                if (stdout.indexOf("Error:  The system cannot find the path specified.") !== -1) {
```

Comment:
Move this out to a const or enum please.

Reply:
Can we not check this against a flag or something instead of the error message? It seems a little wrong to me.